package.json

```diff
@@ -16,7 +16,7 @@
     "@mui/styled-engine-sc": "^5.8.0",
     "@mui/styles": "^5.8.7",
     "@mui/x-date-pickers": "^5.0.0-alpha.7",
-    "@terraware/web-components": "^1.0.18",
+    "@terraware/web-components": "^1.0.19",
```

Comment:
just a request - can we bump this to 20 just to pick up latest change.. and run yarn? otherwise i can send a separate PR but want to avoid a conflict.

Reply:
Sure!